src/services/study/network-modifications.js

```diff
@@ -44,12 +44,14 @@ export function stashModifications(studyUuid, nodeUuid, modificationUuids) {
         encodeURIComponent(studyUuid) +
         '/nodes/' +
         encodeURIComponent(nodeUuid) +
-        '/network-modifications/stash?uuids=' +
+        '/network-modifications' +
```

Comment:
can you use  const urlSearchParams = new URLSearchParams() please to add new params to the url